src/libs/actions/Policy/Policy.ts

```diff
@@ -575,6 +575,10 @@ function clearXeroErrorField(policyID: string, fieldName: string) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {connections: {xero: {config: {errorFields: {[fieldName]: null}}}}});
 }
 
+function clearNetSuiteErrorField(policyID: string, fieldName: string) {
```

Comment:
```suggestion
function clearNetSuiteErrorField(policyID: string, fieldName: string) {function clearConnectionErrorField(policyID: string, connectionName: ConnectionName, fieldName: string) {
    Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {connections: {[connectionName]: {config: {errorFields: {[fieldName]: null}}}}});
}
```

Reply:
I think this comment can be ignored. I saw the type is added to `options.config` for Netsuite. Hence a common method won't work.